dev-python/pypubsub/pypubsub-4.0.3.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{4,5,6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="A Python publish-subcribe library"
+HOMEPAGE="https://github.com/schollii/pypubsub/"
+SRC_URI="
+	https://github.com/schollii/pypubsub/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	"
+
+LICENSE="BSD"
+SLOT="4"
```

Comment:
I don't have any exact info on this, but I believe this should be `0`. Don't change anything for now, but make a plan for a situation in case you get asked to sort these slots.

Reply:
we need slots for Python compatibility.